test/extensions/config_subscription/grpc/delta_subscription_impl_test.cc

```diff
@@ -159,13 +159,15 @@ TEST_P(DeltaSubscriptionNoGrpcStreamTest, NoGrpcStream) {
         std::unique_ptr<Grpc::MockAsyncClient>(async_client), dispatcher, *method_descriptor,
         *stats_store.rootScope(), rate_limit_settings, local_info, false,
         std::make_unique<NiceMock<MockCustomConfigValidators>>(), std::move(backoff_strategy),
-        /*xds_config_tracker=*/XdsConfigTrackerOptRef());
+        /*xds_config_tracker=*/XdsConfigTrackerOptRef(),
```

Comment:
I think given the growth here of names params it might make ense to switch to an options struct. WDYT?

Reply:
Agreed, this will be ~800 lines change. As discussed offline, we'll defer it to a followup PR.